sql/auth/sql_user_table.cc

```diff
@@ -1971,7 +1971,14 @@ static bool acl_tables_setup_for_write_and_acquire_mdl(THD *thd,
          this function.
 */
 
-int open_grant_tables(THD *thd, Table_ref *tables, bool *transactional_tables) {
+#ifdef WITH_WSREP
+int open_grant_tables(THD *thd, Table_ref *tables, bool *transactional_tables,
+                      const char *db __attribute_maybe_unused__,
+                      const char *table __attribute_maybe_unused__) {
```

Comment:
Use the C++17 attribute `[[maybe_unused]]` instead of `__attribute_maybe_unused__`
```suggestion
                      const char *db [[maybe_unused]],
                      const char *table [[maybe_unused]]) {
```